targettests/Remote-Sim/qvector_init_from_state.cpp

```diff
@@ -14,6 +14,9 @@
 // RUN: nvq++ %cpp_std --enable-mlir --target remote-mqpu -fkernel-exec-kind=2 %s -o %t && %t | FileCheck %s
 // clang-format on
 
+// FIXME: state handling with new argument synthesis?
```

Comment:
This is supported actually (this test is aimed to verify the support). Does this test need to be run without `-fkernel-exec-kind=2` , or does it need to be removed and default is used? (see the command line above).

What was the error btw? If the error gives an "unsupported" message, then we must be running old synthesis...

Reply:
Intermittent issue in CI? (This test works fine for me. `shrug`)